vars/hadolintExecute.groovy

```diff
@@ -94,7 +98,11 @@ void call(Map parameters = [:]) {
 
             archiveArtifacts configuration.reportFile
             recordIssues(
-                tools: [checkStyle(name: configuration.reportName, pattern: configuration.reportFile)],
+                tools: [checkStyle(
+                    name: configuration.reportName, 
+                    pattern: configuration.reportFile,
+                    id: "hadolint-"+reportName
```

Comment:
By default this would result into `hadolint-HaDoLint`, maybe we can think of some better naming.

Reply:
Relate this to `dockerFile`.